src/pages/iou/steps/IOUAmountPage.js

```diff
@@ -102,14 +103,14 @@ class IOUAmountPage extends React.Component {
     }
 
     /**
-     * Update amount with number or Backspace pressed.
+     * Update amount with number or Backspace pressed for BigNumberPad.
      * Validate new amount with decimal number regex up to 6 digits and 2 decimal digit to enable Next button
      *
      * @param {String} key
      */
-    updateAmountIfValidInput(key) {
+    updateAmountNumberPad(key) {
         // Backspace button is pressed
-        if (key === '<' || key === 'Backspace') {
+        if (key === '<') {
             if (this.state.amount.length > 0) {
                 this.setState(prevState => ({
                     amount: prevState.amount.substring(0, prevState.amount.length - 1),
```

Comment:
I think this is slightly cleaner (and might fit in one line):

```suggestion
                    amount: prevState.amount.slice(0, -1),
```